### DISCUSSION TITLE
Cascades View Template Body:  Cover stylesheet tiddler

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6772] [Ideas]
I suggest the 

```
$:/config/ViewTemplateBodyFilters/system
```

shall also cover dynamic stylesheet e.g. those tagged with `$:/tags/Stylesheet` but their type is:  `text/vnd.tiddlywiki` or their type is NOT `text/css`

something like this: `[all[current]tag[$:/tags/Stylesheet]!type[text/css]]`

What do you think?

**Comment by pmario** [comment]
So you suggest that tiddlers [listed in the filter](https://tiddlywiki.com/prerelease/#%24%3A%2Fconfig%2FViewTemplateBodyFilters%2Fsystem) should also be displayed as `code`.

I think that would make sense. I'd support it. 

But ... Sometimes I also wished, that there would be a possibility to switch between a `code-body` view and the "rendered-view" ... 

So for the short term, we may add the filter you suggest and for the "long term" we may create a new "feature request" with some new buttons. Where I don't have an idea atm, who the mechanism or the buttons should look like

**Reply by kookma** [reply]
Yes, you are right! in case of dynamic stylesheet, this help a nicer looks and reading them are easier.
I normally go to edit/ preview mode to see how uggly they are ;-)

**Comment by Jermolene** [comment]
Good idea @kookma.

> But ... Sometimes I also wished, that there would be a possibility to switch between a code-body view and the "rendered-view" 

Also a good point.

In Control Panel -> Advanced -> Stylesheets we show the rendered stylesheets as plain text (and not the raw wikitext).

Perhaps we should go further, and make a new template for viewing stylesheet tiddlers that includes tabs for switching between the raw and rendered view? We could add a cascade step for it, and also use it in the control panel listing referred to above.

**Reply by kookma** [reply]
Thank you @Jermolene 
Please let me know if I should convert this to an issue! 

Recently, I ask question/propose suggestion in Discussion to prevent accumulating of issues!

**Reply by pmario** [reply]
Yes. I think both of them can be separate issues. The first one would be a fast short-term fix. The second one, which Jeremy suggests will need more time to be implemented. .... It will be a wikitext only implementation. .. So you could give it a try, if you want.

**Reply by kookma** [reply]
@Jermolene 
Should I submit a PR here: My suggestion at this step is to modify `$:/config/ViewTemplateBodyFilters/system` and add the below filter step

```
[tag[$:/tags/Stylesheet]]
```

**Reply by Jermolene** [reply]
If we weren't going to make the view switchable, then it would be better to use a rendered code view. In other words, like in the control panel, we should show the expanded stylesheet as it will be seen by the browser, rather than the raw wikitext.

**Reply by Jermolene** [reply]
> If we weren't going to make the view switchable, then it would be better to use a rendered code view. In other words, like in the control panel, we should show the expanded stylesheet as it will be seen by the browser, rather than the raw wikitext.

I've created a PR at #6778
